### PR TITLE
More matches

### PR DESCRIPTION
**Pull request by JaceCear**

### CODE REVIEW

File: asm/boss_run.s
Comment (freshollie) [inline asm/boss_run.s:1]: We can remove the rom download from CI ;)

Comment (freshollie) [reply asm/boss_run.s:1]: I can do this in a follow-up

File: include/gba/syscall.h [hunk 1]
```diff
@@ -52,11 +52,19 @@ void RLUnCompVram(const void *src, void *dest);
...
+
+#if PLATFORM_GBA
 s32 Div(s32 num, s32 denom);
 s32 DivArm(s32 denom, s32 num);
-
 s32 Mod(s32 num, s32 denom);
 s32 ModArm(s32 denom, s32 num);
+#else
+// New GCC doesn't like us calling a function 'Mod', so we'll just inline them all.
+#define Div(num, denom)    ({((denom) != 0) ? ((num) / (denom)) : 0;})
```
Comment (freshollie) [inline include/gba/syscall.h:63]: Ah nice

File: include/task.h [hunk 2]
```diff
@@ -104,9 +104,11 @@ struct Task *TaskCreate(TaskMain taskMain, u16 structSize, u16 priority, u16 fla
 
 #define TaskCreate(taskMain, structSize, priority, flags, taskDestructor)               \
     ({                                                                                  \
-        printf("New '%s'(0x%X): 0x%p (in %s,%d)\n", #taskMain, (u32)structSize,         \
-               taskMain, __FILE__, __LINE__);                                           \
-        TaskCreate(taskMain, structSize, priority, flags, taskDestructor, #taskMain);   \
+        struct Task *tt = TaskCreate(taskMain, structSize, priority, flags,             \
+                                     taskDestructor, #taskMain);                        \
+        printf("New '%s' (0x%X, 0x%p) at 0x%p\n", #taskMain, (u32)structSize, taskMain, \
```
Comment (freshollie) [inline include/task.h:109]: Not gonna get a log if this crashes before we log btw, does that help you debugging?

Comment (JaceCear) [reply include/task.h]: Fixed! And yes, it does! :D

**Review by freshollie — APPROVED** [review]

**Pull request merged**